README.md

```diff
@@ -10,4 +10,5 @@ Tools | Description
 ------------ | -------------
 [api-migration](./api-migration) | Facilitates the migration of incident remediation progress across different environments, including SaaS ↔ Self-Hosted, Self-Hosted ↔ Self-Hosted, and SaaS ↔ SaaS.
 [new-arch-migration](./new-arch-migration) | Assists in transitioning from the legacy GitGuardian architecture to the new architecture for Self-Hosted environments.
-[helm-preflights](./helm-preflights) | Ensures GitGuardian requirements are met prior installation or upgrade via [Helm on existing clusters](https://docs.gitguardian.com/self-hosting/installation/installation-existing-helm) by conducting tests from both the local user and the Kubernetes cluster.
\ No newline at end of file
+[helm-preflights](./helm-preflights) | Ensures GitGuardian requirements are met prior installation or upgrade via [Helm on existing clusters](https://docs.gitguardian.com/self-hosting/installation/installation-existing-helm) by conducting tests from both the local user and the Kubernetes cluster.[helm-preflights](./helm-preflights) | Ensures GitGuardian requirements are met prior installation or upgrade via [Helm on existing clusters](https://docs.gitguardian.com/self-hosting/installation/installation-existing-helm) by conducting tests from both the local user and the Kubernetes cluster.
+[honey-token-tools](./helm-preflights) | Script to disseminate honey tokens in your repositories via Pull Requests
```

Comment:
```suggestion
[honey-token-tools](./honey-token-tools) | Script to disseminate [honey tokens](https://docs.gitguardian.com/honeytoken/home) in your repositories via Pull Requests.
```

Reply:
And can we actually rename the folder _honeytoken-tools_ instead of _honey-token-tools_? 🙏